hushline/model.py

```diff
@@ -1,10 +1,13 @@
 import secrets
+from collections import deque
+from contextlib import contextmanager
 from datetime import datetime, timedelta, timezone
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Generator
 
 from flask import current_app
 from flask_sqlalchemy.model import Model
 from passlib.hash import scrypt
```

Comment:
Switch from scrypt to the preferred / recommended argon2.

Reply:
Addressed by (https://github.com/scidsg/hushline/pull/411/commits/6f53d35d4a1c4d29292f3bb7c093187192a02ea9)